lib/node_modules/@stdlib/stats/base/dists/laplace/cdf/benchmark/benchmark.js

```diff
@@ -32,17 +33,25 @@ var cdf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var scale;
+	var len;
 	var mu;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	mu= new Float64Array( len );
+	scale = new Float64Array( len );
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -100.0, 200.0 );
+		mu[ i ] = uniform( -50.0, 100.0 );
```

Comment:
```suggestion
		mu[ i ] = uniform( -50.0, 50.0 );
```

Same comment as above.